.gitignore

```diff
@@ -4,3 +4,6 @@ configs/local.yml
 configs/local.yaml
 **/dotenv
 dev-test/
+.env
+local.mod
+local.sum
```

Comment:
This was an option that I was trying for local development that hasn't quite panned out yet. I haven't found a way to reliably tell editors to use the local module and sum files rather than the main ones when doing local development. I left the exclusions here by accident initially, but I think I'll leave them here anyway. Having extra exclusions for these file names is unlikely to cause problems.